#!/usr/bin/env python
PACKAGE = "behaviors"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("high_goal_speed", double_t, 0, "High Goal Shooting Speed", 0, 0, 27)
gen.add("low_goal_speed", double_t, 0, "Low Goal Shooting Speed", 0, 0, 27)
gen.add("eject_speed", double_t, 0, "Eject Cargo - Shooting Speed", 0, 0, 27)
gen.add("error_margin", double_t, 0, "Shooter margin of error", 0.25, 0, 2)

exit(gen.generate(PACKAGE, "behaviors", "Shooter2022"))
